src/CoreBundle/Resources/config/defaults.inc.php

```diff
@@ -967,7 +967,7 @@
     /**
      * @deprecated since 6.2.0 - no longer used.
```

Comment:
"no longer used" isn't correct anymore

Reply:
Changed to "list caching will be removed in any later Chameleon version."